unique_toolkit/unique_toolkit/chat/functions.py

```diff
@@ -26,20 +151,20 @@ def map_references(references: List[ContentReference]) -> List[Dict[str, Any]]:
     ]
 
 
-def construct_message_modify_params(
-    event_user_id: str,
-    event_company_id: str,
-    event_payload_chat_id: str,
-    event_payload_assistant_message_id: str,
-    event_payload_user_message_id: str,
-    event_payload_user_message_text: str,
+def _construct_message_modify_params(
```

Comment:
Why do we need other add an underscore here, it's not a private method in a class, and it's a utility?